src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsauditapi/resource/AuditResourcePagingTest.kt

```diff
@@ -0,0 +1,134 @@
+package uk.gov.justice.digital.hmpps.hmppsauditapi.resource
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.springframework.beans.factory.annotation.Autowired
+import uk.gov.justice.digital.hmpps.hmppsauditapi.jpa.AuditRepository
+import uk.gov.justice.digital.hmpps.hmppsauditapi.listeners.HMPPSAuditListener.AuditEvent
+import java.time.Instant
+import java.util.UUID
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class AuditResourcePagingTest : IntegrationTest() {
+
+  @Autowired
+  private lateinit var auditRepository: AuditRepository
+
+  val listOfAudits = listOf(
+    AuditEvent(
```

Comment:
Thinking this is more an "e2e test" as I am getting the actual data from the repository...